docs/inversion/depth_of_investigation.ipynb

```diff
@@ -0,0 +1,107 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "c6c8ad14-6e0f-405b-8356-3b9b6ab049f8",
+   "metadata": {},
+   "source": [
+    "# Depth of Investigation\n",
+    "\n",
+    "![masked_model](./images/masked_model.png)\n",
+    "\n",
+    "One way to measure the depth of investigation of a sensitivity based inversion algorithm is to filter the model cells as a percentage (or percentile) of the sensitivity matrix.  SimPEG inversion routines calculate sensitivities at each iteration.  A mask can be calculated at each iteration by asking what cells in the inversion model are associated with a sensitivity that exceeds some threshold.  There are a number of ways to calculate a threshold including percentage, log percentage, and percentile.  In each calculation, the resulting mask should broadly look like a heterogeneous depth cutoff since the data are naturally less sensitive to deeper cells in all geophysical methods. In the following, we discuss the algorithm and implementation of the sensitivity cutoff strategy."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c13ffcec-8d95-4ee9-876f-9aa92e2c534a",
+   "metadata": {},
+   "source": [
+    "## Sensitivities\n",
+    "\n",
+    "The sensitivity matrix is calculated as part of the optimization problem solved by SimPEG while inverting geophysical data.  The sensitivity matrix represents the degree to which each predicted datum changes with respect to a perturbation in each model cell.  It is given in matrix form by\n",
+    "\n",
+    "$$\n",
+    "\\mathbf{J} = \\frac{\\mathbf\\partial{F}(\\mathbf{m})}{\\partial{\\mathbf{m}}}\n",
+    "$$\n",
+    "\n",
+    "where $\\mathbf{m}$ is the model vector, and $\\mathbf{F}(\\mathbf{m})$ represents the forward modelling operation as a function of the model.  The dimensions of the sensitivity are $n\\times m$ where $n$ is the number of data and $m$ are the number of mesh cells.\n",
+    "\n",
+    "The depth of investigation mask is a property of the cells of the mesh only so the rows of the matrix (data) are sum-square normalized as follows.\n",
+    "\n",
+    "$$\n",
+    "\\mathbf{J} = \\Bigg|\\frac{\\mathbf\\partial{F}_{n}(\\mathbf{m})}{\\partial{\\mathbf{m}}}\\Bigg|_{n=1:N}^{2}\n",
```

Comment:
```suggestion
    "\mathbf{doi} = \left[\sum_{n=1}^{N}\left(\frac{\mathbf{J}_{n:}}{w_n}\right)^{2}\right]^{(1/2)}\n",
    
    "where $w_n$ are the data uncertainties associated with the $n^{th}$ datum.
```

Reply:
I'll use $J_norm$ instead of doi since the doi is something calculated later